network-api/networkapi/campaign/views.py

```diff
@@ -125,13 +126,13 @@ def petition_submission(request, petition):
         "email": request.data['email'],
         "email_subscription": request.data['newsletterSignup'],
         "source_url": request.data['source'],
-        "lang": request.data['lang'],
+        "lang": request.data['lang']
```

Comment:
this is python. Ending the last element on a comma is actually a good thing.

Reply:
Sorry about that—I'll revert it.

Reply:
Should there have been errors raised for this when I ran `inv test`?

Reply:
nope, both are fine, but leaving that trailing comma on is "the finer of the two" as far as python is concerned.